tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -179,30 +284,48 @@ private void GSDumpGUI_Load(object sender, EventArgs e)
 
         private void cmdBrowseGSDX_Click(object sender, EventArgs e)
         {
-            FolderBrowserDialog fbd = new FolderBrowserDialog();
-            fbd.Description = "Select the GSdx DLL Directory";
-            fbd.SelectedPath = AppDomain.CurrentDomain.BaseDirectory;
-            if (fbd.ShowDialog() == DialogResult.OK)
-                Settings.GSDXDir = fbd.SelectedPath;
-            Settings.Save();
-            ReloadGsdxDlls();
-
-            // Auto select GSdx dll
-            _availableGsDlls.Selected = _availableGsDlls.Files.FirstOrDefault();
+            OpenFileDialog ofd = new OpenFileDialog();
+            ofd.ValidateNames = false;
+            ofd.CheckFileExists = false;
+            ofd.CheckPathExists = true;
+            ofd.InitialDirectory = Settings.GSDXDir;
+            ofd.FileName = "Select Folder";
+
+            if(ofd.ShowDialog() == DialogResult.OK)
+            {
+                string newpath = Path.GetDirectoryName(ofd.FileName);
+                if (!Settings.GSDXDir.ToLower().Equals(newpath.ToLower()))
```

Comment:
Equals(other, StringComparison.OrdinalIgnoreCase) instead of lowering

Reply:
Done!